src/main/java/com/project/controller/CommentController.java

```diff
@@ -0,0 +1,77 @@
+package com.project.controller;
+
+import com.project.exception.*;
+import com.project.model.request.CommentRequest;
+import com.project.model.response.CommentResponse;
+import com.project.service.CommentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.validation.Valid;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping("api/applications/{appId}/comments")
+public class CommentController {
+    private final CommentService commentService;
+
+    @Autowired
+    public CommentController(CommentService commentService){
+        this.commentService = commentService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<CommentResponse>> fetchComments(@PathVariable("appId") Long appId) {
+        return new ResponseEntity<>(commentService.getAll(appId), HttpStatus.OK);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<CommentResponse> fetchComment(@PathVariable("id") Long id) throws CommentNotFoundException {
+        return new ResponseEntity<>(commentService.getById(id), HttpStatus.OK);
+    }
+
+    @GetMapping(value = "/{id}/attachment/{filename}")
+    public ResponseEntity fetchCommentAttachment(@PathVariable("id") Long id, @PathVariable("filename") String filename)
+            throws CommentNotFoundException, CommentAttachmentNotFoundException {
+        byte[] data = commentService.getAttachment(id, filename);
+        return ResponseEntity.ok().header(HttpHeaders.CONTENT_DISPOSITION,
+                "attachment; filename=\"" + filename + "\"").body(data);
```

Comment:
Maybe you could use `String.format("attachment; filename=\%s\", filename)`;

Reply:
replaced string addition to `String.format()`